plugins/grpc-transport/src/test/java/com/github/bsideup/liiklus/transport/grpc/config/GRPCConfigurationTest.java

```diff
@@ -67,4 +73,27 @@ void shouldValidateParameters() {
         });
     }
 
+    @Test
+    void shouldConsiderTransportConfigurers() {
+        var counter = new AtomicInteger();
+        new ApplicationContextRunner()
+                .withInitializer((ApplicationContextInitializer) new GRPCConfiguration())
+                .withPropertyValues(
+                        "spring.profiles.active: gateway",
+                        "grpc.port: 0"
+                )
+                .withInitializer(ctx -> {
+                    var context = (GenericApplicationContext) ctx;
+                    context.registerBean(LiiklusService.class, () -> Mockito.mock(LiiklusService.class));
+                    context.registerBean(GRPCLiiklusTransportConfigurer.class, () -> builder -> counter.incrementAndGet());
+                })
+                .run(context -> {
+                    assertThat(context)
+                            .hasNotFailed()
+                            .hasSingleBean(GRPCLiiklusService.class)
+                            .hasSingleBean(GRPCLiiklusTransportConfigurer.class);
+                });
+
+        assertThat(counter).describedAs(".apply() called").hasValue(1);
```

Comment:
Consider adding a service or something and then checking that `Server` bean has it, to avoid false positives when configurers are applied on a wrong builder or something